### PR TITLE
EZP-27538 delete translations

### PR DESCRIPTION
**Pull request by mcurran-inviqa**

### CODE REVIEW

File: src/bundle/Resources/views/content/tabs/translations.html.twig [hunk 1]
```diff
@@ -15,10 +19,12 @@
...
-            <td></td>
+            <td>{{ form_widget(actionsForm.removeTranslations[translation.languageCode], {'attr': {'disabled': not translation.canDelete}}) }}</td>
             <td>{{ translation.name }}</td>
             <td>{{ translation.languageCode }}</td>
         </tr>
         {% endfor %}
     </tbody>
 </table>
+
+{{ form_end(actionsForm) }}
```
Comment (maxbaldanza) [inline src/bundle/Resources/views/content/tabs/translations.html.twig]: Can we add a new line at the end please?

File: src/lib/Repository/Values/Content/UiLanguage.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository\Values\Content;
+
+use eZ\Publish\API\Repository\Values\Content\Language;
+
```
Comment (maxbaldanza) [inline src/lib/Repository/Values/Content/UiLanguage.php:10]: Can we add a doc block and `@property-read` for the new variables?

File: src/lib/View/Content/Translations/TranslationParameterSupplier.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\View\Content\Translations;
+
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+use EzSystems\HybridPlatformUi\Repository\UiTranslationService;
+use EzSystems\HybridPlatformUi\View\Content\ParameterSupplier;
+
```
Comment (maxbaldanza) [inline src/lib/View/Content/Translations/TranslationParameterSupplier.php:11]: Can we add a doc block?

File: src/lib/View/Content/Translations/TranslationParameterSupplier.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?php
```
Comment (maxbaldanza) [inline src/lib/View/Content/Translations/TranslationParameterSupplier.php:1]: No Spec?

**Review by maxbaldanza — CHANGES_REQUESTED** [review]
Looks good, just a few doc block issues

File: src/bundle/Resources/views/content/tabs/translations.html.twig [hunk 5]
```diff
@@ -15,10 +19,12 @@
     <tbody>
         {% for translation in translations %}
         <tr>
-            <td></td>
+            <td>{{ form_widget(actionsForm.removeTranslations[translation.languageCode], {'attr': {'disabled': not translation.canDelete}}) }}</td>
```
Comment (maxbaldanza) [inline src/bundle/Resources/views/content/tabs/translations.html.twig:22]: It looks like this is only checked on the front end. Do we not need some server side validation for this?

Comment (mcurran-inviqa) [reply src/bundle/Resources/views/content/tabs/translations.html.twig:22]: the API blocks deletion of the main location and checks user permissions, no need to duplicate before making the call

File: src/lib/Mapper/Form/TranslationMapper.php [hunk 6]
```diff
@@ -0,0 +1,26 @@
+<?php
```
Comment (maxbaldanza) [inline src/lib/Mapper/Form/TranslationMapper.php:1]: Can we add license and doc block to this?

**Review by maxbaldanza — CHANGES_REQUESTED** [review]
Looks good apart from this small change 👍

**Pull request closed without merging**